File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/orca/ORCALibGetDocInfo.java
package nta.med.data.model.ihis.orca;

public class ORCALibGetDocInfo {
	private String yoyangGiho       ;
	private String yoyangName       ;
	private String zipCode          ;
	private String address           ;
	private String tel               ;
	private String presidentName    ;
	private String orcaGigwanCode  ;
	public ORCALibGetDocInfo(String yoyangGiho, String yoyangName,
			String zipCode, String address, String tel, String presidentName,
			String orcaGigwanCode) {
		super();
		this.yoyangGiho = yoyangGiho;
		this.yoyangName = yoyangName;
		this.zipCode = zipCode;
		this.address = address;
		this.tel = tel;
		this.presidentName = presidentName;
		this.orcaGigwanCode = orcaGigwanCode;
	}
	public String getYoyangGiho() {
		return yoyangGiho;
	}
	public void setYoyangGiho(String yoyangGiho) {
		this.yoyangGiho = yoyangGiho;
	}
	public String getYoyangName() {
		return yoyangName;
	}
	public void setYoyangName(String yoyangName) {
		this.yoyangName = yoyangName;
	}
	public String getZipCode() {
		return zipCode;
	}
	public void setZipCode(String zipCode) {
		this.zipCode = zipCode;
	}
	public String getAddress() {
		return address;
	}
	public void setAddress(String address) {
		this.address = address;
	}
	public String getTel() {
		return tel;
	}
	public void setTel(String tel) {
		this.tel = tel;
	}
	public String getPresidentName() {
		return presidentName;
	}
	public void setPresidentName(String presidentName) {
		this.presidentName = presidentName;
	}
	public String getOrcaGigwanCode() {
		return orcaGigwanCode;
	}
	public void setOrcaGigwanCode(String orcaGigwanCode) {
		this.orcaGigwanCode = orcaGigwanCode;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Orca/ORCALibGetDocInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Orca
{
    [Serializable]
    public class ORCALibGetDocInfo
    {
        private String _yoyangGiho;
        private String _yoyangName;
        private String _zipCode;
        private String _address;
        private String _tel;
        private String _presidentName;
        private String _orcaGigwanCode;

        public String YoyangGiho
        {
            get { return this._yoyangGiho; }
            set { this._yoyangGiho = value; }
        }

        public String YoyangName
        {
            get { return this._yoyangName; }
            set { this._yoyangName = value; }
        }

        public String ZipCode
        {
            get { return this._zipCode; }
            set { this._zipCode = value; }
        }

        public String Address
        {
            get { return this._address; }
            set { this._address = value; }
        }

        public String Tel
        {
            get { return this._tel; }
            set { this._tel = value; }
        }

        public String PresidentName
        {
            get { return this._presidentName; }
            set { this._presidentName = value; }
        }

        public String OrcaGigwanCode
        {
            get { return this._orcaGigwanCode; }
            set { this._orcaGigwanCode = value; }
        }

        public ORCALibGetDocInfo() { }

        public ORCALibGetDocInfo(String yoyangGiho, String yoyangName, String zipCode, String address, String tel, String presidentName, String orcaGigwanCode)
        {
            this._yoyangGiho = yoyangGiho;
            this._yoyangName = yoyangName;
            this._zipCode = zipCode;
            this._address = address;
            this._tel = tel;
            this._presidentName = presidentName;
            this._orcaGigwanCode = orcaGigwanCode;
        }

    }
}